pyt/usage.py

```diff
@@ -91,7 +91,21 @@ def _add_optional_group(parser):
         action='store_true',
         help='do not skip lines with # nosec comments'
     )
-
+    optional_group.add_argument(
+        '-r', '--recursive', dest='recursive',
+        action='store_true', help='find and process files in subdirectories'
+    )
+    optional_group.add_argument(
+        '-x', '--exclude',
+        dest='excluded_paths',
+        action='store',
+        default='',
+        help='Separate files with commas'
+    )
+    optional_group.add_argument(
```

Comment:
I guess targets will be part of `_add_required_group` b/c it's replacing `-f files`